spec/_includes/publish-box.html

```diff
@@ -0,0 +1,19 @@
+
+
+<p id="publish-box">
+This is the Continuous Integration Build of the {{site.data.fhir.igName}} Implementation Guide, based on <a href="{{ site.data.fhir.path }}">FHIR Version {{ site.data.fhir.version }}</a>. See the <a href="{{base}}history/index.html">Directory of published versions <img src="external.png"/></a>
+</p>
+
+<!--
+
```

Comment:
If this commented code is no longer needed, can you remove?

Reply:
I like keep this to switch back to CI build text - since usually have both a CI Build and a publish build, but OK with removing.